packages/process/_test.pony

```diff
@@ -31,16 +31,12 @@ class iso _TestStdinStdout is UnitTest
     let notifier: ProcessNotify iso = _ProcessClient(size, "", 0, h)
     try
       let path = FilePath(h.env.root as AmbientAuth, "/bin/cat")?
-      let args: Array[String] iso = recover Array[String](1) end
-      args.push("cat")
-      let vars: Array[String] iso = recover Array[String](2) end
-      vars.push("HOME=/")
-      vars.push("PATH=/bin")
+      let args: Array[String] val = [ "cat" ]
+      let vars: Array[String] val = [ "HOME=/"; "PATH=/bin" ]
```

Comment:
It's a nitpick, but the whitespace here doesn't match the style mentioned [here in the style guide for this project](https://github.com/ponylang/ponyc/blob/master/STYLE_GUIDE.md#array-literals).

Reply:
😃 Thank you! I was vaguely remembering something, but didn't look hard enough. I'll fix these soon.

Reply:
(But _soon_ means tomorrow.)

Reply:
⏩ #2956